README.rst

```diff
@@ -0,0 +1,143 @@
+universal\_notifications
+========================
+|travis|_ |pypi|_ |codecov|_
+
+**High-level framework for notifications**
+
+This project is intended to provide a convenient way to send notifications using multiple
+notification backends (e.g., e-mail, SMS, push).
+
+--------------
+
+Setting up
+----------
+
+To start using **universal\_notifications** please add ``universal_notifications`` to
+``INSTALLED_APPS`` in your Django project, and then migrate the app:
+``./manage.py migrate universal_notifications``.
+
+Basic usage
+-----------
+
+-  `WebSocket notifications`_
+-  `E-mail notifications`_
+-  `SMS notifications`_
+-  `Push notifications`_
+
+WebSocket notifications
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To have Universal Notifications receive WS notifications (ie. to mark notification as received)
+add to your settings.py:
+
+::
+
+    WS4REDIS_SUBSCRIBER = 'universal_notifications.backend.websockets.RedisSubscriber'
+
+Simple example of using WS notifications:
+
+.. code:: python
+
+    class OrderShippedWS(WSNotification):
+        message = 'order_shipped'
+        serializer_class = OrderSerializer
+
+    # ... somewhere in a view
+    OrderShippedWS(item=order, receivers=[user], context={}).send()
+
+E-mail notifications
+~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    class OrderShippedEmail(EmailNotification):
+        email_name = 'order_shipped'
+        email_subject = _('{{user.full_name}}, Your order no. {{item.pk}} has been shipped.')
```

Comment:
I've corrected the doc, but yeah it'd be nice to have the context passed to the template.